docs/realm.js

```diff
@@ -231,6 +231,17 @@ class Realm {
      * @returns {true} if compaction succeeds.
      */
     compact() {}
+
+    /**
+     * If the Realm is a partially synchronized Realm, fetch and synchronize the objects
+     * of a given object type that match the given query (in string format).
+     *
+     * **Partial synchronization is a tech preview. Its APIs are subject to change.**
+     * @param {Realm~ObjectType} type - The type of Realm objects to retrieve.
+     * @param {string} query - Query used to filter objects.
+     * @param {callback} - function to call when objects are created, modified, or deleted.
+     */
+    subscribeToObjects(className, query, callback) {}
```

Comment:
Should this return a promise rather than taking a callback?

The callback taken by `partial_sync::register_query` is called precisely once: when the server has computed the initial result set, or determined that the query had an error. It's not called again when the result set changes.